apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/AppDocumentListAdapter.java

```diff
@@ -28,13 +28,14 @@
     @NonNull
     @Override
     public AppDocumentListHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
-        return new AppDocumentListHolder(LayoutInflater.from(mContext).inflate(R.layout.item_view_app, null, false));
+        return new AppDocumentListHolder(LayoutInflater.from(mContext).inflate(R.layout.item_view_app, viewGroup, false));
     }
 
     @Override
-    public void onBindViewHolder(@NonNull AppDocumentListHolder holder, final int position) {
+    public void onBindViewHolder(@NonNull AppDocumentListHolder holder, @SuppressLint("RecyclerView") final int position) {
```

Comment:
what is that warning you suppressed?

Reply:
Fair point. I copypasted it from anotehr plae for no good reason.